lib/flight_job/commands/create_script.rb

```diff
@@ -109,6 +150,27 @@ def prompt_answers
       def prompt
         @prompt = TTY::Prompt.new(help_color: :yellow)
       end
+
+      # Technically multiple flags may try and read STDIN. Whilst this would be an "unusual"
+      # use case, it is still "technically" valid. In this case STDIN becomes the input for
+      # both flags. However as the input can only be read once, it needs to be cached
+      def cached_stdin
+        @cached_stdin ||= $stdin.read_nonblock(FlightJob.config.max_stdin_size).tap do |str|
+          if str.length == FlightJob.config.max_stdin_size
+            raise InputError, "The STDIN exceeds the maximum size of: #{FlightJob.config.max_stdin_size}B"
+          end
+        end
+      rescue Errno::EWOULDBLOCK, EOFError
+        raise InputError, "Failed to read the data from the Standard Input"
```

Comment:
Remove the capitalisation of standard input please.

Reply:
c2968f93078c9098653107f48c025a1bf2112d29